app/channel.go

```diff
@@ -1198,3 +1198,19 @@ func GetPinnedPosts(channelId string) (*model.PostList, *model.AppError) {
 		return result.Data.(*model.PostList), nil
 	}
 }
+
+func GetDirectChannel(userId1, userId2 string) (*model.Channel, *model.AppError) {
```

Comment:
No action required, functionally it's the same as `CreateDirectChannel()` but it does the DB read first which is probably less strain on the database. Might be better to combine them in the future but doesn't have to block this PR